cli/ingress-controller/main.go

```diff
@@ -282,8 +282,8 @@ func main() {
 		controllerConfig.Kong.InMemory = true
 	}
 	if kongDB == "cassandra" {
-		log.Warn("running controller with kong cassandra as a datastore is deprecated; " +
-			"please consider using postgres or in-memory mode")
+		log.Fatalf("Cassandra-backed deployments of Kong managed by the ingress controller are no longer supported;" +
```

Comment:
Nit: all errors should start with lower case letter.

Reply:
Should be fine here--guidance is that it's fine if the leading word is a proper noun, and Cassandra is: https://github.com/golang/go/wiki/CodeReviewComments#error-strings